packages/angular_devkit/architect/BUILD

```diff
@@ -12,28 +12,19 @@ package(default_visibility = ["//visibility:public"])
 ts_library(
     name = "architect",
     srcs = glob(
-        ["src/**/*.ts"],
-        # Currently, this library is used only with the rollup plugin.
```

Comment:
why remove the comment? was it stale?

Reply:
The comment is not applicable at all. It looks like it was copy pasted from the build-optimizer BUILD file.